nipype/algorithms/confounds.py

```diff
@@ -299,35 +301,41 @@ def _run_interface(self, runtime):
     def _list_outputs(self):
         return self._results
 
+
 class CompCorInputSpec(BaseInterfaceInputSpec):
     realigned_file = File(exists=True, mandatory=True,
-        desc='already realigned brain image (4D)')
-    mask_file = InputMultiPath(File(exists=True, deprecated='0.13',
-        new_name='mask_files',
-        desc='One or more mask files that determines ROI (3D)'))
-    mask_files = InputMultiPath(File(exists=True,
-        desc='One or more mask files that determines ROI (3D)'))
+                          desc='already realigned brain image (4D)')
+    mask_files = InputMultiPath(File(exists=True),
+                                desc=('One or more mask files that determines '
+                                      'ROI (3D)'))
     merge_method = traits.Enum('union', 'intersect', 'none', xor=['mask_index'],
-        requires=['mask_files'],
-        desc='Merge method if multiple masks are present - `union` aggregates '
-        'all masks, `intersect` computes the truth value of all masks, `none` '
-        'performs CompCor on each mask individually')
-    mask_index = traits.Range(0, xor=['merge_method'], requires=['mask_files'],
-        desc='Position of mask in `mask_files` to use - first is the default')
+                               requires=['mask_files'],
+                               desc=('Merge method if multiple masks are '
+                                     'present - `union` aggregates all masks, '
+                                     '`intersect` computes the truth value of '
+                                     'all masks, `none` performs CompCor on '
```

Comment:
What about

`union` uses voxels included in at least one input mask, `intersect` uses only voxels present in all input masks